content/learning-paths/cross-platform/simd-on-rust/conclusion.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Conclusion
+weight: 7
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+You have seen a few aspects of writing SIMD code on Arm with Rust. In the end which is better, C or Rust?
+
+Performance wise there is no difference really, as you have seen. Rust is perfectly capable of generating the same assembly code as C in most cases.
+
+If you want to program optimal SIMD code using the Arm ASIMD/Neon intrinsics, then `std::arch` is the obvious choice.
+
+If however, your approach needs to be as portable as possible and you don't want to spend time providing multiple implementations for each architecture, then `std::simd` is also starting to become a very viable alternative, even though it's not part of the stable compiler yet.
+
+So in the end it is mostly about your personal preference, Rust is a modern language and provides many new features that are hard to be ignored even by the most hardcore C and C++ developers. It is possible to write safe code with C and C++, but it requires more effort and more tools, whilst Rust provides that by default.
+
+### What about SVE, SVE2 or even SME?
+
+What we have mentioned so far applies only to ASIMD/Neon, as it is a "traditional" SIMD architecture. With the recent addition of SVE and its newer sibling SVE2 or the future SME (Scalable Matrix Extension), things become quite more complicated.
```

Comment:
I would remove the bits about `"traditional" SIMD architecture` and reword `things become quite more complicated` a bit.. maybe `a different approach is required`.